core/src/main/java/com/sequenceiq/cloudbreak/controller/validation/template/TemplateValidator.java

```diff
@@ -29,13 +29,15 @@
 import com.sequenceiq.cloudbreak.domain.Template;
 import com.sequenceiq.cloudbreak.domain.VolumeTemplate;
 import com.sequenceiq.cloudbreak.domain.stack.Stack;
+import com.sequenceiq.cloudbreak.domain.stack.instance.InstanceGroup;
 import com.sequenceiq.cloudbreak.dto.credential.Credential;
 import com.sequenceiq.cloudbreak.validation.ValidationResult;
 import com.sequenceiq.cloudbreak.workspace.model.User;
 import com.sequenceiq.common.api.type.CdpResourceType;
 
 @Component
 public class TemplateValidator {
+    private static String groupNameIDBroker = "idbroker";
```

Comment:
This can be `final` and by convention `static final` variables are upper cased.

https://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html

>If your variable stores a constant value, such as static final int NUM_GEARS = 6, the convention changes slightly, capitalizing every letter and separating subsequent words with the underscore character. By convention, the underscore character is never used elsewhere.